app/controllers/admin/users_controller.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+module Admin
+  class UsersController < ApplicationController
+    # rubocop:disable Rails/LexicallyScopedActionFilter
+    before_action :set_user, only: %i[show edit update destroy]
+    # rubocop:enable Rails/LexicallyScopedActionFilter
+
+    def index
+      @users = User.all
```

Comment:
aqui tah fazendo varias queries na tabela stores, uma para cada user que tem uma store, n+1, vc pode usar o includes

```suggestion
      @users = User.includes(:stores)
```